fastparquet/util.py

```diff
@@ -106,3 +106,13 @@ def index_like(index):
                 index._start == 0 and
                 index._stop == len(index) and
                 index._step == 1 and index.name is None)
+
+
+def check_column_names(columns, *args):
+    """Ensure that parameters listing column names have corresponding columns"""
+    for arg in args:
+        if isinstance(arg, (tuple, list)):
```

Comment:
Only `index=`, which will fail in the allocation stage in fastparquet, and is treated specially in dask anyway. I didn't want to check non-list values, since some parameters can take simple values like True, False, 'infer', which are then applied to all appropriate columns.